apps/api/test/services/wallet-testing.service.ts

```diff
@@ -27,6 +43,82 @@ export class WalletTestingService {
     return { user, token };
   }
 
+  /**
+   * Creates a registered user and returns the user and token.
+   * Specify the user code to use for the user.
+   * "debug" will create a user with the email "dev@example.com" and the nickname "dev".
+   * "debug1" will create a user with the email "dev1@example.com" and the nickname "dev1".
+   * "debug2" through "debug20" will create users with corresponding emails and nicknames.
+   * This is setup in the docker-compose.dev.yml file.
+   * @param userCode - The user code to use for the user.
+   * @returns The user and token.
+   */
+  async createRegisteredUser(
+    userCode:
+      | "debug"
+      | "debug1"
+      | "debug2"
+      | "debug3"
+      | "debug4"
+      | "debug5"
+      | "debug6"
+      | "debug7"
+      | "debug8"
+      | "debug9"
+      | "debug10"
+      | "debug11"
+      | "debug12"
+      | "debug13"
+      | "debug14"
+      | "debug15"
+      | "debug16"
+      | "debug17"
+      | "debug18"
+      | "debug19"
+      | "debug20" = "debug"
+  ) {
+    const tokenResponse = await axios.post(
```

Comment:
suggestion(non-blocking): I think you can just use `fetch` API. It should be available in testEnv = node